packages/vats/src/lib-chainStorage.js

```diff
@@ -54,7 +52,11 @@ export function makeChainStorageRoot(
   function makeChainStorageNode(path) {
     const node = {
       getStoreKey() {
-        return handleStorageMessage({ key: path, method: 'getStoreKey' });
+        return handleStorageMessage({
+          key: path,
+          method: 'getStoreKey',
+          value: '',
```

Comment:
We shouldn't need to send `value` for "getStoreKey" messages.
```suggestion
```

Reply:
without it the payload doesn't satisfy the `StorageMessage` type, which is defined to match Golang. I could make that `Partial<>` but I figure it's better to reveal the JS devs that Golang will always interpret some string

Reply:
OK, that's fair.